apps/aehttp/src/aehttpc_aeternity.erl

```diff
@@ -163,19 +194,45 @@ post_pin_tx(SignedSpendTx, NodeSpec) ->
     %lager:debug("PINNING: wrote to PC tx hash: ~p", [TxHash]),
     encode_child_pin_payload(TxHash).
 
+pin_contract_call(ContractPubkey, PinTx, Who, Amount, Fee, SignModule) ->
+    Nonce = get_local_nonce(Who),
+    {ok, CallData} = aeb_fate_abi:create_calldata("pin", [PinTx]),
+    ABI = 8, % not really nice, what is the supported version of getting the latest ABI version
+    TxSpec =
+        #{  caller_id   => aeser_id:create(account, Who)
+          , nonce       => Nonce
+          , contract_id => aeser_id:create(contract, ContractPubkey)
+          , abi_version => ABI
+          , fee         => Fee
+          , amount      => Amount
+          , gas         => 10
```

Comment:
gas can be a reasonable smallish number like 1000 or so, normally it won't use a lot.